Defra.Cdp.Backend.Api/appsettings.json

```diff
@@ -8,41 +8,6 @@
     "DatabaseUri": "mongodb://cluster.address/admin?authSource=$external&authMechanism=MONGODB-AWS",
     "DatabaseName": "cdp-portal-backend"
   },
-  "Github": {
-    "Organisation": "DEFRA",
-    "Scheduler": {
-      "quartz.scheduler.instanceName": "Github Scheduler for fetching repositories",
-      "quartz.threadPool.maxConcurrency": 4
-    },
-    "PollIntervalSecs": 60,
-    "AppKey": "supersecretbase64encodedpem",
-    "AppId": 1,
-    "AppInstallationId": 1,
```

Comment:
I think we still need this bit for pulling the repo info from github

Reply:
Ah, maybe Rider got a bit too over-excited. Will revert.